alonzo/impl/src/Cardano/Ledger/Alonzo/Data.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE EmptyDataDeriving #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Cardano.Ledger.Alonzo.Data
+  ( Data,
+    EraIndependentData,
+    DataHash (..),
+    hashData,
+  )
+where
+
+import Cardano.Binary (ToCBOR (..))
+import qualified Cardano.Crypto.Hash as Hash
+import Cardano.Ledger.Crypto (HASH)
+import Cardano.Ledger.Era (Crypto, Era)
+import Control.DeepSeq (NFData)
+import GHC.Generics (Generic)
+import NoThunks.Class (NoThunks)
+import Shelley.Spec.Ledger.Hashing (HashAnnotated (..))
+
+-- | TODO this should be isomorphic to the plutus type
+data Data era
```

Comment:
Should it be imported from the Plutus library? Did we ever conclude the discussion on how close this is going to be to plain CBOR?

Reply:
I think Plutus will have or already has a particular single module that (re?)-exports everything that should be accessible to the ledger (this type, the interpreter calling function, etc.)